core/src/wallet/cosmos/CosmosLikeAccount.hpp

```diff
@@ -150,6 +154,7 @@ namespace ledger {
                         private:
                                 std::shared_ptr<CosmosLikeAccount> getSelf();
 
+                                std::shared_ptr<cosmos::Account> _accData;
```

Comment:
I usually find it more readable to use full words for variables and functions, as long as they reasonably fit on the screen, but REALLY a personal taste matter so I won't insist on this one ^^